src/stores/notificationStore.ts

```diff
@@ -0,0 +1,49 @@
+import { action, computed, observable } from 'mobx';
+
+interface INotification {
+    message: string;
+    type: string;
+}
+
+export class NotificationStore {
+    @observable private _notifications: INotification[];
+
+    constructor() {
+        this._notifications = [];
+    }
+
+    @computed
+    get notifications(): INotification[] {
+        return this._notifications;
+    }
+
+    @action
+    public addNotifications(notifications: INotification[]) {
```

Comment:
Maybe give one notification at once instead of an array?
-> public addNotification(notification: INotification) { }

its rare that we would add multiple notifications and if needed, we can call this multiple times